flink-python/pyflink/datastream/tests/test_data_stream.py

```diff
@@ -589,6 +589,40 @@ def process_element2(self, value, ctx: 'CoProcessFunction.Context'):
         side_expected = ['0', '0', '1', '1', '2', '3']
         self.assert_equals_sorted(side_expected, side_sink.get_results())
 
+    def test_co_broadcast_side_output(self):
+        tag = OutputTag("side", Types.INT())
+
+        class MyBroadcastProcessFunction(BroadcastProcessFunction):
+
+            def process_element(self, value, ctx):
+                yield value[0]
+                yield tag, value[1]
+
+            def process_broadcast_element(self, value, ctx):
+                yield value[1]
+                yield tag, value[0]
+
+        self.env.set_parallelism(2)
```

Comment:
This is unnecessary as the parallelism is 2 by default. Refer to PyFlinkStreamingTestCase for more details.

Reply:
This is kind of an "explict reminder" that tells we expect the output result should match parallelism=2 with some elements duplicated.